R/postprocess.R

```diff
@@ -1,3 +1,132 @@
+#' Extract samples of the delay distribution parameters
+#'
+#' @param fit A model fit with `epidist::epidist`
+#' @param ... Additional arguments passed to `brms::prepare_predictions`
+#' @family postprocess
+#' @autoglobal
+#' @export
+delay_samples <- function(fit, ...) {
```

Comment:
As we are living in cmdstanr and posterior world I see the argument for using draws

Reply:
I think one of the `R CMD CHECK` warnings is that we import `posterior` but don't use it. So unclear how much we are living in `posterior` world

Reply:
I think an argument for draws is that is sounds less like a function. Samples sounds like it's doing something (more than draws somehow).

Reply:
I think that's also an argument for the "extract" type word. To make it clear that sampling is not happening here, it's just manipulation of samples.